languages/csharp/exercises/concept/casting/.docs/instructions.md

```diff
@@ -0,0 +1,86 @@
+Our football club (first encountered in (cross-ref-tba)) is soaring in the leagues, and you have been invited to do some more work, this time on the laminate printing system.
```

Comment:
This could totally be me, but I had to google what laminate printing was :)

Reply:
I have changed it to "security pass".

Laminate is an appealing example of a metonym.

Reply:
Aha, okay.